### ISSUE TITLE
Accessibility Violations in Web component - Multiple elements with “search” role do not have unique labels

### ISSUE THREAD

**Issue by lbenicio-ibm** [carbon-design-system/carbon-components-vue#1389]
## Detailed description
Accessibility Violations in Web component reported by IBM Equal Access Accessibility Checker - Multiple elements with “search” role do not have unique labels

> Describe in detail the issue you're having.

the vue web component cv-data-table have multiple html tags with role="search" which casuses a flag on accessibility checker

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> Is this issue related to a specific component?

cv-data-table

> What did you expect to happen? What happened instead? What would you like to see changed?

Role value for input field inside search box should be "searchbox" not "search"

> What browser are you working in?

Safari

> What version of the Carbon Design System are you using?

10.46.0

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

eConfig

## Steps to reproduce the issue

1. Create a vue component cv-data-table
2. enable search on that table
3. run accessibility checker

## Additional information

- Screenshots or code
- Notes
